src/screens/dashboard/dashboard.tsx

```diff
@@ -259,6 +259,7 @@ export class DashboardScreenComponent extends React.Component<
                                     account={this.props.selectedAccount}
                                     blockchain={blockchain}
                                     navigation={this.props.navigation}
+                                    showBottomPadding={blockchains.length > 1}
```

Comment:
why not show all the time this padding?

Reply:
Because if there is only one active Blockchain, the bottom blockchain nav is hidden and the bottom padding will show like this:

![image](https://user-images.githubusercontent.com/32679726/74633738-fced1300-516a-11ea-8b9a-84b7c8ddedb7.png)

If you consider that it's unnecessary, it can be easily removed.